tests/db.cpp

```diff
@@ -17,14 +19,32 @@ TEST(DB, EmptyDatabase) {
     auto db = DB::Create(":memory:");
 
     const size_t UNKNOWN_ID = 100;
-    EXPECT_THAT(db->GetProducts(), testing::IsEmpty());
-    EXPECT_THAT(db->GetTableware(), testing::IsEmpty());
+
+    auto products = std::move(db->GetProducts());
```

Comment:
std::move is redundant here - remove from all similar places (I will not comment on this any more)